docs/style_guide/operators_keywords_and_boundaries.md

```diff
@@ -0,0 +1,104 @@
+# Operators, Keywords and Boundaries
+
+## Single line formatting
+It is recommend to follow the following rules when applying white spaces on a line. 
+Following will list down where a single whitespace can apply.
+### Keywords
+* Keywords should always followed by a single space. Exceptions are as below.
+    - For types like map which followed by `<>`
+    - This will be ignore when defining rest parameter (`string...`).
+    - If record type has a rest field no spaces between type and rest operator(`...`) 
+    ```ballerina
+    type RecordName2 record {
```

Comment:
Use a meaningful record name ex: Person, then the example would be more descriptive and cleaner

Reply:
Will add the change